include/deal.II/lac/solver_gcr.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include <deal.II/lac/solver.h>
+
+namespace dealii
```

Comment:
There are a lot of issues with these first five lines of the file :-)

Reply:
There is a reason why the PR is WIP ;)